.circleci/config.yml

```diff
@@ -527,6 +527,24 @@ jobs:
           command:
             python3 CircleciScripts/build_iossample.py "$(pwd)/aws-sdk-ios-samples"
   
+  release_xcframeworks:
+    macos:
+      xcode: "12.3.0"
+    steps:
+      - skip_job_unless_required
+      - checkout_with_cache
+      - pre_start_simulator
+      - restore_cache:
+          key: build_xcframeworks-result-{{ .Revision }}
+      - run:
+          name: Prepare xcframework archives
+          command:
+            python3 CircleciScripts/prepare_xcframework_archives.py $release_version
+      - run:
+          name: Upload xcframework archives
+          command:
+            python3 CircleciScripts/upload_xcframeworks.py $release_bucket $release_version sdk_xcf_release
+
   release_ios_s3:
```

Comment:
suggest rename to `release_xcframeworks_combined` or similar

Reply:
Renamed